java/vector/src/main/codegen/templates/UnionReader.java

```diff
@@ -39,7 +39,7 @@
 @SuppressWarnings("unused")
 public class UnionReader extends AbstractFieldReader {
 
-  private BaseReader[] readers = new BaseReader[45];
+  private BaseReader[] readers = new BaseReader[46];
```

Comment:
It'd be good if this constant was given a name.

Reply:
Changed